data/styles/clear/include/Basemap.mapcss

```diff
@@ -433,9 +433,16 @@ area|z10-[landuse=farmyard],
 
 /* Parking */
 
-area|z15-[amenity=parking]
+area|z15-[amenity=parking],
 {fill-color: @parking;}
 
+area|z15-[amenity=parking][parking=underground],
```

Comment:
- it doesn't work (you can see areas are not removed in drules*.txt)
- change to `[amenity=parking][location=underground]` - should work
- that's because kothic pocesses only the first matching rule from mapcss-mapping.csv, i.e. in `[amenity=parking][location=underground],[amenity=parking][parking=underground]` it ignores everything after comma (multiple matching rules are used by the generator only)

Reply:
ah i didn't know that, fixed :)